File: Aula01/app/src/main/java/com/wolneyhqf/aula01/MainActivity.java
package com.wolneyhqf.aula01;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    private EditText campoDeTextNome;
    private Button botaoEnviar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        campoDeTextNome = (EditText) findViewById(R.id.campo_de_texto_nome);
        botaoEnviar = (Button) findViewById(R.id.botao_enviar);

        Intent intent = new Intent(Intent.ACTION_CALL);

        botaoEnviar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String nome = campoDeTextNome.getText().toString();
                //Toast.makeText(getBaseContext(), "Seu nome é: " + nome, Toast.LENGTH_LONG).show();v
                Intent intent = new Intent(getBaseContext(), SegundaActivity.class);
                Bundle bundleIntent = new Bundle();
                bundleIntent.putString("nome_digitado", nome);
                intent.putExtras(bundleIntent);
                startActivity(intent);
            }
        });

        Log.i("AppAula", "MainActivity.onCreate()");
    }

    @Override
    protected void onStart(){
        super.onStart();
        Log.i("AppAula", "MainActivity.onStart()");
    }

    @Override
    protected void onResume(){
        super.onResume();
        Log.i("AppAula", "MainActivity.onResume()");
    }

    @Override
    protected void onPause(){
        super.onPause();
        Log.i("AppAula", "MainActivity.onPause()");
    }

    @Override
    protected void onStop(){
        super.onStop();
        Log.i("AppAula", "MainActivity.onStop()");
    }

    @Override
    protected void onRestart(){
        super.onRestart();
        Log.i("AppAula", "MainActivity.onRestart()");
    }

    @Override
    protected void onDestroy(){
        super.onDestroy();
        Log.i("AppAula", "MainActivity.onDestroy()");
    }

}

File: 2018/AulaActivity-22-09/app/src/main/java/br/edu/unifavip/aulaactivity/MainActivity.kt
package br.edu.unifavip.aulaactivity

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.util.Log
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    val context: Context = this

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main_multiplicacao)

        Log.d("msg", "MainActivity.onCreate()")

        val buttonCalcular = findViewById<Button>(R.id.buttonCalcular)

        buttonCalcular.setOnClickListener{
            val editTextNumero1 = findViewById<EditText>(R.id.editTextNumero1)
            val editTextNumero2 = findViewById<EditText>(R.id.editTextNumero2)

            var numero1 = 0.0
            if(editTextNumero1.text.toString() != ""){
                numero1 = editTextNumero1.text.toString().toDouble()
            }

            var numero2 = 0.0
            if(editTextNumero2.text.toString() != ""){
                numero2 = editTextNumero2.text.toString().toDouble()
            }

            var resultado = numero1 * numero2
            Toast.makeText(context, "Resultado: $resultado", Toast.LENGTH_SHORT).show()

            val minhaIntent = Intent(context, SegundaActivity::class.java)

            minhaIntent.putExtra("resultado", resultado)

            startActivity(minhaIntent)
        }

    }

    override fun onStart(){
        super.onStart()
        Log.d("msg", "MainActivity.onStart()")
    }

    override fun onResume(){
        super.onResume()
        Log.d("msg", "MainActivity.onResume()")
    }

    override fun onPause(){
        super.onPause()
        Log.d("msg", "MainActivity.onPause()")
    }

    override fun onStop(){
        super.onStop()
        Log.d("msg", "MainActivity.onStop()")
    }

    override fun onRestart(){
        super.onRestart()
        Log.d("msg", "MainActivity.onRestart()")
    }

    override fun onDestroy(){
        super.onDestroy()
        Log.d("msg", "MainActivity.onDestroy()")
    }

}
